packages/one-app-runner/bin/one-app-runner.js

```diff
@@ -72,6 +71,8 @@ const createYargsConfig = () => {
     })
     .option('modules', {
       type: 'array',
+      // eslint-disable-next-line global-require
+      demandOption: !require('yargs').argv.moduleMapUrl,
```

Comment:
Can you add a test?

Reply:
Technically this test is already covering that.
```
test('command errors out if --module-map-url option is not given', () => {
  const consoleErrorSpy = jest.spyOn(console, 'error');

  process.argv = ['', '', '--root-module-name', 'frank-lloyd-root', '--docker-image', 'one-app:5.0.0'];
  jest.mock('../../src/startApp', () => jest.fn());
  require('../../bin/one-app-runner');
  expect(consoleErrorSpy.mock.calls).toMatchSnapshot();
});
```

But I can add something like
```
test('--modules option is required if --module-map-url option is not given', () => {
  const consoleErrorSpy = jest.spyOn(console, 'error');

  process.argv = ['', '', '--root-module-name', 'frank-lloyd-root', '--docker-image', 'one-app:5.0.0'];
  jest.mock('../../src/startApp', () => jest.fn());
  require('../../bin/one-app-runner');
  expect(consoleErrorSpy.mock.calls).toMatchSnapshot();
});
```

Reply:
Added in https://github.com/americanexpress/one-app-cli/pull/48/commits/e0828338471fc7749b49630e11e78688e7b29967